packages/x-charts/src/models/axis.ts

```diff
@@ -209,7 +209,26 @@ export type AxisConfig<S extends ScaleName = ScaleName, V = any> = {
    * The key used to retrieve `data` from the `dataset` prop.
    */
   dataKey?: string;
+  /**
+   * Formats the axis value.
+   * @param {V} value The value to format.
+   * @returns {string} The string to display.
+   */
   valueFormatter?: (value: V) => string;
+  /**
+   * Formats the axis value to be displayed in axis ticks.
+   * If defined, this formatter overrides the `valueFormatter` for ticks.
+   * @param {V} value The value to format.
+   * @returns {string} The string to display.
+   */
+  tickValueFormatter?: (value: V) => string;
```

Comment:
I'm hesitating with 
- `tickFormatter` which is shorter and should not interfere with others
- `valueTickFormatter` which has the advantage to group all the formatters at the same place in the alphabetical order

Reply:
This is an interesting situation. 🤔 
`tickValueFormatter` seems like it makes the most sense, but have you considered going the other way - adding a context to the existing `valueFormatter`? 🤔 

```ts
interface ValueFomatterContext {
  location: 'value' | 'tick' | 'tooltip';
  // or
  location?: 'tick' | 'tooltip';
}
///
valueFormatter?: (value: V, context: ValueFomatterContext) => string;
```

Reply:
I did not had preferences between the two, but effectively it fixes the naming issue :+1: